docs/README.bridge.md

```diff
@@ -8,6 +8,8 @@ This device plugin allows user to attach pods to existing bridges on nodes.
 
 ## Usage
 
+_The plugin must be run in the kube-system namespace._
```

Comment:
@gageorsburn we need to update configmap configuration in this README too, it does not specify namespace.

Reply:
@phoracek Good catch. Namespaces updated in both places.

Reply:
@gageorsburn still failing:

```
I0815 12:29:20.189786    3268 reflector.go:240] Listing and watching *v1.ConfigMap from github.com/kubevirt/kubernetes-device-plugins/pkg/network/bridge/lister.go:85
E0815 12:29:20.192274    3268 reflector.go:205] github.com/kubevirt/kubernetes-device-plugins/pkg/network/bridge/lister.go:85: Failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:kube-system:default" cannot list configmaps in the namespace "kube-system"
```

Are you sure kube-system does not need any RBAC configuration? Maybe there is a different trick there, if not, we can drop the Namespace requirement and create a new Role.

Reply:
@phoracek That's really odd. I just tested with a fresh Minikube install and no issues.

Log: https://pastebin.com/MFL9HwEL

In what conditions are you trying to run it? I'm curious at this point and would like to better understand the RBAC nuances.

Reply:
@gageorsburn does Minikube have RBAC enabled?

Build DP image and sync it with local cluster:

```
make build-network-bridge docker-build-network-bridge cluster-down cluster-up cluster-sync-network-bridge
```

Create daemon set with the DP:

```
./cluster/kubectl.sh create -f ds.yml
```

ds.yml:

```yaml
apiVersion: apps/v1
kind: DaemonSet
metadata:
  labels:
    name: device-plugin-network-bridge
  name: device-plugin-network-bridge
  namespace: kube-system
spec:
  selector:
    matchLabels:
      name: device-plugin-network-bridge
  template:
    metadata:
      labels:
        name: device-plugin-network-bridge
    spec:
      hostNetwork: true
      hostPID: true
      containers:
      - name: device-plugin-network-bridge
        image: registry:5000/device-plugin-network-bridge
        securityContext:
          privileged: true
        volumeMounts:
          - name: var-run
            mountPath: /var/run
          - name: device-plugin
            mountPath: /var/lib/kubelet/device-plugins
      volumes:
        - name: var-run
          hostPath:
            path: /var/run
        - name: device-plugin
          hostPath:
            path: /var/lib/kubelet/device-plugins
```

And finally read logs using `./cluster/kubectl.sh logs`.

Reply:
It did yeah, I explicitly passed in the RBAC flag. You can see it in the log.

```
 gageorsburn@mbp  ~/go/src/github.com/kubevirt/kubernetes-device-plugins   autoconfigmap  minikube start --feature-gates=DevicePlugins=true --extra-config=apiserver.authorization-mode=RBAC
```

I'll try that and see what happens.

Reply:
Yeah that's strange. I'm not sure why minikube with RBAC enabled is different than the Makefile cluster. On minikube the default namespace would give me this same error but kube-system did not. Would you rather us create a new service account or just give kube-system default read-only permissions?

`create rolebinding default-view --clusterrole=view --serviceaccount=kube-system:default --namespace=kube-system`

@phoracek